.circleci/config.yml

```diff
@@ -1,6 +1,8 @@
 version: 2.1
 
 orbs:
+  browser-tools: circleci/browser-tools@1.3.0
```

Comment:
With this PR, I started getting ""Unable to find Chrome executeable" errors from CI. I still have to investigate when this issue started to occur. However, added this orb and installing ChromeDriver fixes this issue.